src/content/docs/en/guides/upgrade-to/v6.mdx

```diff
@@ -371,6 +371,38 @@ prefetch('/about');
 
 <ReadMore>Learn more about [prefetching](/en/guides/prefetch/).</ReadMore>
 
+### Removed: `rewrite()` from Actions context
+
+<SourcePR number="14477" title="feat!: remove rewrite from action context"/>
+
+In Astro 5.5.6, `rewrite()` on Actions context was deprecated as it was not meant to be exposed here.
```

Comment:
```suggestion
In Astro 5.5.6, the `ActionAPIContext.rewrite` method was deprecated as it was not meant to be exposed here.
```
I traced back to that original PR, and there wasn't any reason/description given there either, but in an ideal world we would be able to say something about what using this feature was/is... like, "Actions handlers were not intended to allow serving content from another page ..." (or explain why a rewrite in an action handler isn't safe, doesn't make sense... or often when we deprecate something it's in favour of something else.

 Do you know what I mean? Something that describes maybe how or when this might have been (improperly?) used that caused us to discover it was there and needed to remove it?

Reply:
I think the reason is that actions use endpoints under the hood, but you're not supposed to have access to so much power here. If you need a rewrite, use an endpoint basically. Maybe @ematipico has more context

Reply:
OK, I'll take more context if we have it and Ema has a great explanation, but non-blocking. I'd *prefer* something other than "oops, shouldn't have been here" like, "you were supposed to achieve this with X instead..." or "you should never do this because..."

I'll let you decide whether having something like that is reasonable, or you just want to merge it as at least correct upgrade instructions!

Reply:
What about something like this?
```suggestion
In Astro 5.5.6, the `ActionAPIContext.rewrite` method was deprecated as it was not meant to be exposed here. You should custom endpoints you need rewrites instead.
```